.eslintrc.js

```diff
@@ -43,7 +43,7 @@ module.exports = {
     {
       // turned off otherwise non-admin modules will complain about importing components from admin
       // via the novel silverstripe js component sharing setup
-      'import/no-extraneous-dependencies': [
```

Comment:
`import/no-extraneous-dependencies` is no longer the warning being thrown, so would need to change it to `import/no-unresolved`

All of the `// eslint-disable-next-line import/no-extraneous-dependencies` scattered out through the codebase were probably not doing much, so I've removed them all

Reply:
> so I've removed them all

As in there are separate PRs for this in silverstripe/admin, etc? Where are those PRs? Or did I misunderstand this statement?